modules/virt-sno-differences.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-sno-differences_{context}"]
+= {sno-caps} {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster.
+
+When provisioning an {sno} cluster with the assisted installer, preconfigured persistent storage is deployed automatically.
+
+* In {VirtProductName} 4.10 and 4.11, the HostPath Provisioner (HPP) is automatically installed.
+* In {VirtProductName} 4.12, the {product-title} Data Foundation Logical Volume Manager Operator is the provided out-of-the-box storage solution. You can also manually deploy using the HPP.
```

Comment:
This now renders as "OpenShift Container Platform Data Foundation Logical Volume Manager Operator", which isn't correct. This one of those cases where it's fine to hard code "OpenShift" because it's part of the Operator name: "OpenShift Data Foundation Logical Volume Manager Operator"

Reply:
Gotcha, thanks.